index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="assets/responsive.css" />
+    <link
+      href="https://unpkg.com/boxicons@2.0.9/css/boxicons.min.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div id="container">
+      <header>
+        <nav>
+          <p id="logo">Lisandro Seia</p>
+
+          <ul>
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <section id="present">
```

Comment:
- [ ] The section should have the correct background image. 
## You Design
![image](https://user-images.githubusercontent.com/18730106/143321094-78eec9a7-4933-4403-a2d1-f71b2a2a5399.png)

## Figma Design 
![image](https://user-images.githubusercontent.com/18730106/143321072-b545b3a9-1e55-4b85-ba1a-12d802947c1f.png)

- [ ] When you implement this; make sure to link the correct background image using CSS and not using a regular `img` tag.